### PR TITLE
Unpin isort and upgrade packages

### PR DESCRIPTION
**Pull request by cpennington**

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -102,7 +102,6 @@ pymongo==3.8.0            # via edx-opaque-keys
 pyopenssl==19.0.0         # via ndg-httpsclient, paypalrestsdk
 python-dateutil==2.8.0
 python-openid==2.2.5 ; python_version == "2.7"
-python3-openid==3.1.0 ; python_version >= "3"  # via social-auth-core
```
Comment (crice100) [inline requirements/base.txt]: Should this have been removed?

**Review by crice100 — APPROVED** [review]
See question about _python3-openid_

File: requirements/base.txt (hunk 1 above)
Comment (cpennington) [reply requirements/base.txt]: I'm not sure. Given that the social-auth-core version number didn't change, probably not, I'll put it back.

File: requirements/production.txt [hunk 2]
```diff
@@ -107,7 +107,6 @@ pyopenssl==19.0.0         # via ndg-httpsclient, paypalrestsdk
 python-dateutil==2.8.0
 python-memcached==1.58
 python-openid==2.2.5 ; python_version == "2.7"
-python3-openid==3.1.0 ; python_version >= "3"  # via social-auth-core
```
Comment (crice100) [inline requirements/production.txt]: Did we want to put this back, too?

Comment (cpennington) [reply requirements/production.txt]: oops. yup.

**Pull request merged**